exec/csnex/Cell.cs

```diff
@@ -431,6 +470,15 @@ public void Set(Object o)
             Object = o;
         }
 
+        public void Set(object o)
```

Comment:
I'm not sure it's wise to have an overload for the generic C# `object`. Doing so means that any call to `Set` where you pass anything that doesn't match any other overload, ends up here even if you didn't mean to. I would prefer to see `SetObject` here (and corresponding named methods for other cell types). There is no need for overloading here.

Reply:
The next pull request will revamp the `csnex.Cell` class.